docs/modules/ROOT/pages/vshn-managed/postgresql/create.adoc

```diff
@@ -38,6 +38,19 @@ spec:
 
 NOTE: To get more information about all available configuration options, please see the xref:references/crds.adoc#k8s-api-github-com-vshn-component-appcat-apis-vshn-v1-vshnpostgresql[API Reference]
 
+=== Sidecar Overhead and Fair-Use Policy
+
+Each PostgreSQL Standalone instance has following overhead:
+
+* CPU: 364m
+* Memory: 704Mi
+
+These are for various sidecar containers that run alongside the actual PostgreSQL instance.
+These containers handle connection pooling, metrics and other internal functionality.
+
+NOTE: Please be aware that the normal https://docs.appuio.cloud/user/explanation/fair-use-policy.html[Fair Use Policy] applies to AppCat Services by VSHN as well.
```

Comment:
```suggestion
NOTE: Please be aware that the normal https://docs.appuio.cloud/user/explanation/fair-use-policy.html[Fair Use Policy] applies to AppCat Services by VSHN as well.
```
NOTE: Please be aware that https://docs.appuio.cloud/user/explanation/fair-use-policy.html[Fair Use Policy] applies to all AppCat Services by VSHN.

Reply:
I'm not sure if I understood the last part. I changed a bit but seems the meaning is different. You might want to take a look again